src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,116 @@
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  min-width: 1024px;
```

Comment:
this does not prevent design to break on wider screens )
<img width="936" alt="wider screen" src="https://user-images.githubusercontent.com/39308717/200787984-173827ba-b31e-4258-b69c-53a985adead3.png">

so i guess a hardcoded width would do just fine )
```suggestion
  width: 1024px;
```